composer.json

```diff
@@ -9,7 +9,7 @@
     "spryker/application": "^3.0.0",
     "spryker/glue-application": "^1.27.0",
     "spryker/guzzle": "^2.1.0",
-    "spryker/kernel": "^3.53.0",
+    "spryker/kernel": "^3.30.0 as 3.53.0",
```

Comment:
Keep here ^3.53.0

And instead on project level alias it
`dev-master as 3.53.0`

that might work better.